docs/en/docs/advanced/response-directly.md

```diff
@@ -10,6 +10,12 @@ But you can return a `JSONResponse` directly from your *path operations*.
 
 It might be useful, for example, to return custom headers or cookies.
 
+# Return a Model directly
+
+In case you're already initializing a `pydantic`'s `BaseModel`, you can return `ModelResponse` directly to increase performance
+by skipping re-validation and calls to `jsonable_encoder` by FastAPI.
```

Comment:
why **"re"** validation ? Should it be **" by skipping validation and calls to `jsonable_encoder` by FastAPI."**
	
nothing protect from this case

```python
@app.get("/", response_model=Model)
async def get_pydantic() -> ModelResponse:
    a = Model(arg_str="toto", arg_int = 5)
    a.arg_int="tata"
    return ModelResponse(a)
```

Reply:
Done.